MockKitUI/View Controllers/MockCGMManagerSettingsViewController.swift

```diff
@@ -467,6 +474,10 @@ final class MockCGMManagerSettingsViewController: UITableViewController {
                 vc.contextHelp = "The sine curve parameters describe a mathematical model for glucose value production."
                 vc.delegate = self
                 show(vc, sender: sender)
+            case .scenario:
+                cgmManager.dataSource.model = .scenario(pastSamples: [], futureSamples: [])
+                tableView.reloadRows(at: indexPaths(forSection: .model, rows: ModelRow.self), with: .automatic)
+                NotificationCenter.default.post(name: .init("showScenarioSelector"), object: nil)
```

Comment:
Not a fan of this "secret handshake" undocumented api call. As I said in the dev sync, I don't think we really need this feature.

Reply:
Sounds good!